### PR TITLE
Update 04-expertise.md

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR makes several changes to episode 04-expertise.md, including:

1) Updating objectives, key points and questions to stick to a small number of each and fit the new episode contents.

2) Changing the episode title to focus on expertise (there is no discussion of memory in this episode anymore).

3) Adding explicit examples of how the limitations of expertise affect teaching in our workshop setting. 

4) Move content about dismissive language from motivation section because it ties in well with discussion of expert blind spot. 

5) Add exercises for understanding limitations of expertise and developing strategies to overcome them while teaching. 

6) Reiterate importance of formative assessment and explicitly say how it can help overcome limitations of expertise.

### CODE REVIEW

File: _episodes/04-expertise.md [hunk 1]
```diff
@@ -1,103 +1,120 @@
...
-- "This reinforcement often takes the form of formative assessment."
-- "One way concept maps can help in lesson planning is by defining appropriate points in a lesson for formative assessment."
+- "With practice, we can learn to overcome our expert blind spot."
 ---
 
 We now discuss what distinguishes expertise
 from earlier stages of learning, how being an expert can make it 
 more difficult to teach novices, and some tools to help instructors 
-identify their own expert blind spots.  
+identify overcome these difficulties.
```
Comment (ChristinaLK) [inline _episodes/04-expertise.md]: +identify and overcome?

File: _episodes/04-expertise.md [hunk 2]
```diff
@@ -1,103 +1,120 @@
...
-identify their own expert blind spots.  
+identify overcome these difficulties.  
 
-## Expertise
+## What Makes an Expert?  
 
-An [earlier topic]({{ page.root }}/01-practice.md) described a key 
+An [earlier topic]({{ page.root }}/02-practice.md) described a key 
 difference between novices and competent practitioners. Competent practitioners 
 have mental models that work well enough for most situations.
```
Comment (ChristinaLK) [inline _episodes/04-expertise.md]: Add a note about novices: "Novices lack a mental model, and competent practitioners have..."
OR
Remove the "Competent practitioners have..." sentence.  

This sets up the "how are experts different from both" question more clearly.

File: _episodes/04-expertise.md [hunk 3]
```diff
@@ -1,103 +1,120 @@
...
-> of mental models - a mental model is a way to facilitate making connections between
-> separate facts.
+> of mental models - a mental model is (in part) a set of connections or relationships
+> among facts or concepts.
 {: .callout}
 
-> ## Other Consequences of Expertise
+## Limitations of Expertise  
+
+Because your learners' mental models will be less densly connected than your own,
```
Comment (ChristinaLK) [inline _episodes/04-expertise.md]: Not all over our instructors are experts -- maybe add a "likely" or "probably"?  That waters down the language but...

Also, densly -> densely

File: _episodes/04-expertise.md [hunk 4]
```diff
@@ -1,103 +1,120 @@
...
+> Take a few minutes to plan out how you would explain that error message to your learners. Write the 
+> error and your explaination in the Etherpad.
+> 
+> This discussion should take about 5 minutes.  
+{: .challenge}
 
-In fact, experts are frequently so familiar with their subject
+Experts are frequently so familiar with their subject
 that they can no longer imagine what it's like to *not* see the world that way. 
 This is called *expert blind spot* and can lead to what's known as the *expertise-reversal effect*.
```
Comment (ChristinaLK) [inline _episodes/04-expertise.md]: Based on this text, it's not clear to me what the expertise reversal effect is.

File: _episodes/04-expertise.md [hunk 5]
```diff
@@ -111,15 +128,54 @@ reason why we welcome instructors who still identify as "novices"!  Someone
...
+>
+> What other words or phrases can have the effect of demotivating learners? What alternatives can we use to
+> express this meaning in a positive and motivational way?  
+>
+> In the Etherpad, make a list of demotivating words/phrases and alternatives.  
+>
+> This discussion should take about 5 minutes.  
+>
+>> ## Solution  
+>> Courtney Seiter lists [10 words and phrases][motivation-words] that can change a conversation: if, could, yes, together, thank you,
```
Comment (ChristinaLK) [inline _episodes/04-expertise.md]: italicize the words for emphasis?

File: _episodes/04-expertise.md [hunk 6]
```diff
@@ -150,43 +206,25 @@ For these reasons,
 we have adopted a "teach most immediately useful first" approach
 described in [this episode]({{ page.root }}/19-motivation/).
 
-> ## Software Carpentry Is Not Computer Science
+> ## Software/Data Carpentry Is Not Computer Science  
 >
 > Many of the foundational concepts of computer science,
 > such as computability,
-> inhabit the lower-right corner of the grid described above.
+> are diffult to learn and not immediately useful.
```
Comment (ChristinaLK) [inline _episodes/04-expertise.md]: diffult -> difficult

**Comment by ErinBecker** [comment]
Thanks for the thorough review @ChristinaLK! I think I've addressed all your comments, including the addition of how we frame asking for questions. Great addition.

**Comment by ErinBecker** [comment]
Merged at approval from @ChristinaLK

**Pull request merged**